src/classes/storage.ts

```diff
@@ -78,10 +88,10 @@ export class Storage {
     const groundPlace: GroundPlace | undefined = this.groundPlaces[placeIndex];
 
     if (!groundPlace || groundPlace.type !== GroundPlaceType.CLUSTER) {
-      throw new Error(`The StopCluster with the Gpuid ${stopClusterGpuid} is not found.`);
+      throw new Error(`The StopCluster with the Gpuid "${stopClusterGpuid}" is not found.`);
     }
 
-    return groundPlace;
+    return JSON.parse(JSON.stringify(groundPlace));
```

Comment:
is it because of typing that you have to do this ?

Reply:
Nope, actually this is to not get the groundPlace by its reference and don't modify it through this `get` method

Reply:
Solved with **cloneDeep** from **lodash.cloneDeep**